go-controller/observability-lib/sampledecoder/nb_client.go

```diff
@@ -0,0 +1,110 @@
+package sampledecoder
```

Comment:
This module is more than just a `nb_client`.  It also configures the OVS database.  Should it be called `db_client.go`?